content/docs/command-reference/repro.md

```diff
@@ -51,7 +51,8 @@ There are a few ways to restrict what will be regenerated by this command: by
 specifying specific reproduction [`targets`](#options), or by using certain
 command [options](#options), such as `--single-item` or `--all-pipelines`.
 
-> Note that stages without dependencies are considered _always changed_, so
+> Note that stages without dependencies are considered
+> [always changed](/doc/command-reference/stage/add#--always-changed), so
 > `dvc repro` always executes them.
```

Comment:
Hmmm Idk. `stage add --always-changed` adds an `always-changed` field to the stage in dvc.yaml . That's not needed for stages without dependencies to be considered always changed. This link may end up causing confusion. What was the motivation for this change? Maybe I can help clarify the text in another way.

Reply:
Maybe linking to https://dvc.org/doc/command-reference/status#local-workspace-status could help more in clarifying... 🤔